src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub struct Method {
 }
 
 pub trait Validate {
-    fn validate(&self, module: &str, method: &str) -> Result<(), &'static str> {
+    fn validate(&self, _module: &str, _method: &str) -> Result<(), &'static str> {
         Ok(())
     }
```

Comment:
Could u remove `Ok(())` and make the signature like `fn validate(&self, module: &str, method: &str) -> Result<(), &'static str>;` (which I think is the correct syntax for trait methods)

Reply:
Sounds good. This is providing a default implementation but now that I look at it, it's not used anywhere